benchmarks/test_peer_selection.py

```diff
@@ -22,81 +22,78 @@
     absolute_import, unicode_literals, print_function, division
 )
 
-import mock
-import random
+from tornado import gen, ioloop
```

Comment:
yes they both are using this benchmark. I compared the mash branch and the
heap branch. The different between these two branch is just the peer heap
change. In order to make it exact apple to apple comparison , we also
change the peer choose in mash branch to choose the least pending request's
peer so keep them using the same branch.
On Tue, Jan 12, 2016 at 11:17 AM Bryce Lampe <notifications@github.com>
wrote:

> In benchmarks/test_peer_selection.py
> <https://github.com/uber/tchannel-python/pull/355#discussion_r49499585>:
>
> > @@ -22,81 +22,78 @@
> >      absolute_import, unicode_literals, print_function, division
> >  )
> >
> > -import mock
> > -import random
> > +from tornado import gen, ioloop
>
> Are the perf numbers in the summary master vs. this branch running old vs.
> new benchmark or are they both running this benchmark? Need
> apples-to-apples comparisons.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/uber/tchannel-python/pull/355/files#r49499585>.
>